src/pages/admin/Event/EventStats/EventStatsTable.js

```diff
@@ -272,6 +272,9 @@ export class EventStatsTable extends Component {
   registrationNumbers(users) {
     const registrationNumbers = {
     };
+    for (let i = 0; i < users.length; i++) {
+      users = users.filter(user => user.isPartner === false);
+    }
```

Comment:
this can be rewritten the same if you remove the for loop. Notice how you are not using "i" in line 276. See the documentation for array.filter to see how filter operates on the entire array and not just a single item.